doc/dev/release.md

```diff
@@ -307,11 +310,31 @@ OPERATORSUM="$(sha256sum operator-sdk | cut -d ' ' -f 1)"
 brew bump-formula-pr --strict --url=$OPERATORSDKURL --sha256=$OPERATORSUM operator-sdk
 ```
 
-Note: If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
+**NOTE**  If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
```

Comment:
It doesn't look like this was resolved. And actually for consistency with the rest of the doc's notes, I'd suggest this:

```suggestion
**Note:**  If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
```